src/Microsoft.AspNetCore.SignalR.Core/HubCallerContext.cs

```diff
@@ -8,20 +8,44 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A context abstraction for accessing information about the hub caller.
+    /// </summary>
     public abstract class HubCallerContext
     {
+        /// <summary>
+        /// Gets the caller connection ID.
```

Comment:
Let's not repeat 'caller' in all these. The type is already about the 'caller' :). We can put "connection" in if we want to have a little more clarity